pkg/antctl/raw/multicluster/create/access_token.go

```diff
@@ -15,186 +15,96 @@
 package create
 
 import (
-	"context"
 	"fmt"
+	"os"
 	"strings"
 
 	"github.com/spf13/cobra"
-	corev1 "k8s.io/api/core/v1"
-	rbacv1 "k8s.io/api/rbac/v1"
-	"k8s.io/apimachinery/pkg/api/errors"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/client-go/rest"
-	"sigs.k8s.io/controller-runtime/pkg/client"
-
-	"antrea.io/antrea/pkg/antctl/raw"
-	multiclusterscheme "antrea.io/antrea/pkg/antctl/raw/multicluster/scheme"
+
+	"antrea.io/antrea/pkg/antctl/raw/multicluster/common"
 )
 
-type accessTokenOptions struct {
+type memberTokenOptions struct {
 	namespace      string
 	serviceAccount string
 	roleBinding    string
+	output         string
 }
 
-var accessTokenOpts *accessTokenOptions
+var memberTokenOpts *memberTokenOptions
 
-var accessTokenExamples = strings.Trim(`
-# Create an access token in a leader cluster for one or more member clusters. If the ServiceAccount or RoleBinding does not exist, the command will create them too
-  $ antctl mc create accesstoken <NAME> -n <NAMESPACE> --service-account <SERVICE_ACCOUNT> --role-binding <ROLE_BINDING>
+var memberTokenExamples = strings.Trim(`
+# Create a member token.
+  $ antctl mc create membertoken member-access-token -n antrea-multicluster
+# Create a member token with specified ServiceAccount and RoleBinding.
+  $ antctl mc create membertoken member-access-token -n antrea-multicluster --service-account member-sa --role-binding member-rb
+# Create a member token and save it into a file.
+  $ antctl mc create membertoken member-access-token -n antrea-multicluster --service-account member-sa --role-binding member-rb --output token.yaml
 `, "\n")
 
-func (o *accessTokenOptions) validateAndComplete() error {
+func (o *memberTokenOptions) validateAndComplete() error {
 	if o.namespace == "" {
-		return fmt.Errorf("the Namespace cannot be empty")
-	}
-	if o.serviceAccount == "" {
-		return fmt.Errorf("the ServiceAccount cannot be empty")
-	}
-	if o.roleBinding == "" {
-		return fmt.Errorf("the RoleBinding cannot be empty")
+		return fmt.Errorf("the Namespace is required")
 	}
 	return nil
 }
 
 func NewAccessTokenCmd() *cobra.Command {
 	command := &cobra.Command{
-		Use:     "accesstoken",
+		Use:     "membertoken",
 		Args:    cobra.MaximumNArgs(1),
-		Short:   "Create an accesstoken in a leader cluster",
-		Long:    "Create an accesstoken in a leader cluster",
-		Example: accessTokenExamples,
-		RunE:    accessTokenRunE,
+		Short:   "Create a membertoken in a leader cluster",
+		Long:    "Create a member token in a leader cluster, which will be saved in a Secret. A ServiceAccount and a RoleBinding will be created too if they do not exist.",
+		Example: memberTokenExamples,
+		RunE:    memberTokenRunE,
 	}
 
-	o := &accessTokenOptions{}
-	accessTokenOpts = o
+	o := &memberTokenOptions{}
+	memberTokenOpts = o
 	command.Flags().StringVarP(&o.namespace, "namespace", "n", "", "Namespace of the ClusterClaim")
```

Comment:
"Antrea Multi-cluster Namespace".

Or if you prefer change the description for commands in leader (init, destrory, create membertoken) to "Namespace of the ClusterSet", and use "Antrea Multi-cluster Namespace" for the member commands.

Reply:
Sure. Changed the description in leader command to "Namespace of the ClusterSet" and the description in member commands to "Antrea Multi-cluster Namespace"